src/push/fcm.rs

```diff
@@ -68,10 +59,25 @@ pub struct MessageResult {
     pub error: Option<String>,
 }
 
+#[derive(Debug)]
+pub struct FcmStateConfig {
```

Comment:
What is a "state config"? From looking at the usages, this seems to be the API key plus the endpoint. None of that is stateful. Why not just `FcmConfig` (and `HmsConfig` etc)? Or maybe `FcmEndpointConfig`?

Reply:
I called it this way to differentiate it from the config that is parsed from the config file, and I wasn't happy with the naming, too... FcmConfig is fine for me if you don't mind the same name

Reply:
renamed to FcmConfig